zeebe/broker/src/main/java/io/camunda/zeebe/broker/exporter/context/ExporterContext.java

```diff
@@ -37,7 +40,18 @@ public ExporterContext(
     this.logger = logger;
     this.configuration = configuration;
     this.partitionId = partitionId;
-    this.meterRegistry = meterRegistry;
+    underlyingMetricRegistry = meterRegistry;
+    this.meterRegistry = new CompositeMeterRegistry();
+    // meterRegistry is null in tests
+    if (meterRegistry != null) {
+      this.meterRegistry.add(meterRegistry);
```

Comment:
:x: Should be 
```suggestion
      this.meterRegistry.add(underlyingMetricRegistry);
```

Reply:
Let's add a test for that too, checking that metrics published by an exporter are forwarded to the underlying registry

Reply:
Ah no wait, this is correct because `meterRegistry` is the param, not the field. My bad!

Test could be useful anyway.

Reply:
I'll add assertions that in both registry we can see the same metrics